packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -2499,6 +2499,33 @@ it('should change page size', async () => {
   expect(rows[49].querySelector('.iui-cell')?.textContent).toEqual('Name50');
 });
 
+it('should render number of rows selected for paginator', async () => {
+  const { container } = renderComponent({
+    data: mockedData(100),
+    pageSize: 10,
+    paginatorRenderer: (props) => <TablePaginator {...props} />,
+    isSelectable: true,
+  });
+
+  const rowCheckboxes = container.querySelectorAll(
+    '.iui-table-body .iui-row .iui-checkbox',
+  );
+
```

Comment:
Nit picking but here are too many new lines 😀 Try grouping related code with new lines, like you click something and then expect some result.

Reply:
The new lines have been removed